jni/lib/src/jreference.dart

```diff
@@ -27,31 +23,41 @@ extension ProtectedJReference on JReference {
 /// A class which holds one or more JNI references, and has a `delete` operation
 /// which disposes the reference(s).
 abstract class JReference implements Finalizable {
-  //TODO(PR): Is it safe to cast void *f (void *) to void f (void *)?
   static final _finalizer =
       NativeFinalizer(Jni.env.ptr.ref.DeleteGlobalRef.cast());
 
-  JReference.fromRef(this.reference) {
-    _finalizer.attach(this, reference, detach: this);
+  JReference.fromRef(this._reference) {
+    _finalizer.attach(this, _reference, detach: this);
   }
 
   bool _deleted = false;
 
   /// Check whether the underlying JNI reference is `null`.
-  bool get isNull => reference == nullptr;
+  bool get isNull => _reference == nullptr;
```

Comment:
Should this be `reference` or `_reference`?

Reply:
@dcharkes

Reply:
I guess it doesn't really matter to check it here as well, so I'll go with `_reference`.

Reply:
Looking from the tests, the uses actually are null (for example in the list test). If we don't want these references to change value over time (also not by deleting), I'd say it should be `reference`.